doc/reference/autoinstall-reference.rst

```diff
@@ -323,7 +322,7 @@ can be expressed in two different ways:
 * The special value ``country-mirror``
 * A mapping with the following keys:
 
-  * ``uri``: The URI of the mirror to use, e.g., "http://fr.archive.ubuntu.com/ubuntu"
+  * ``uri``: The URI of the mirror to use, e.g., `fr.archive.ubuntu.com/ubuntu <http://fr.archive.ubuntu.com/ubuntu>`_
```

Comment:
This is always specified in URI form.  We're not trying to link to the France mirrors - probably better that it not be a link at all actually, we're specifying the URI value, so the previous text was more correct.

Reply:
Ack